spring-cloud-aws-autoconfigure/src/test/java/io/awspring/cloud/autoconfigure/config/parameterstore/ParameterStoreConfigDataLoaderIntegrationTests.java

```diff
@@ -70,7 +70,7 @@ class ParameterStoreConfigDataLoaderIntegrationTests {
 
 	@Container
 	static LocalStackContainer localstack = new LocalStackContainer(
-			DockerImageName.parse("localstack/localstack:1.4.0")).withServices(SSM).withReuse(true);
+			DockerImageName.parse("localstack/localstack:2.0.0")).withServices(SSM).withReuse(true);
```

Comment:
There should be no neeed anymore to set the list of services when creating LocalStackContainer.